src/FixedDataTable.js

```diff
@@ -1239,9 +1239,9 @@ var FixedDataTable = createReactClass({
         var availableScrollWidth = adjustedWidth - totalFixedColumnsWidth;
 
         // Get width of specified column
-        var selectedColumnWidth = columnInfo.bodyScrollableColumns[
+        var selectedColumnWidth =columnInfo.bodyScrollableColumns.length >0? columnInfo.bodyScrollableColumns[
           scrollableColumnIndex
-        ].props.width;
+        ].props.width:100;
```

Comment:
Does the value used here matter?  If not can you use 0 instead of 100 to avoid magic numbers.

Reply:
Hi, 

I think it is an edge case; when there is no state ( and so no columns etc) and then indicating scrollable columns.

I have added an if statement at the beginning of the block (if it is OK).

Spacing and the magic number should be OK, as I removed them.

Thanks for the great work.